<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tela_Cadastro_Livro</class>
 <widget class="QMainWindow" name="Tela_Cadastro_Livro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>152</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Título:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="titulo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>183</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dados do livro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_salvar_livro">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>340</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>SALVAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/save_file_disk_open_searsh_loading_clipboard_1513.png</normaloff>images/save_file_disk_open_searsh_loading_clipboard_1513.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>218</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Autor: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="autor">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>249</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="qtd_pag">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>182</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>217</y>
      <width>169</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ano de publicação:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ano_publi">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>248</y>
      <width>188</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_selecionar_img">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SELECIONAR IMAGEM</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/imageup_imagen_12892.png</normaloff>images/imageup_imagen_12892.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="colocar_imagem">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>951</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantidade de páginas:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>751</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ISBN: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="isbn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QToolButton" name="buttonVoltar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/back_12955.png</normaloff>images/back_12955.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <zorder>textBrowser</zorder>
   <zorder>label_2</zorder>
   <zorder>textBrowser_3</zorder>
   <zorder>label_7</zorder>
   <zorder>titulo</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>qtd_pag</zorder>
   <zorder>autor</zorder>
   <zorder>ano_publi</zorder>
   <zorder>botao_salvar_livro</zorder>
   <zorder>botao_selecionar_img</zorder>
   <zorder>textBrowser_2</zorder>
   <zorder>label_5</zorder>
   <zorder>label_4</zorder>
   <zorder>isbn</zorder>
   <zorder>colocar_imagem</zorder>
   <zorder>buttonVoltar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMOSTRAR_LIVROS_CADASTRADOS">
   <property name="text">
    <string>MOSTRAR LIVROS CADASTRADOS</string>
   </property>
  </action>
  <action name="actionBUSCAR">
   <property name="text">
    <string>BUSCAR</string>
   </property>
  </action>
  <action name="actionEDITAR">
   <property name="text">
    <string>EDITAR</string>
   </property>
  </action>
  <action name="actionLISTAR">
   <property name="text">
    <string>LISTAR LIVROS CADASTRADOS</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
